src/elements/element.point.js

```diff
@@ -72,6 +73,32 @@ module.exports = function(Chart) {
 			ctx.lineWidth = helpers.getValueOrDefault(vm.borderWidth, globalOpts.elements.point.borderWidth);
 			ctx.fillStyle = vm.backgroundColor || defaultColor;
 
+			// Cliping for Points.
+			var isOutofChartArea = function(position, area, errMarginPercent) {
+				if ((position.x < area.left) || (area.right*errMarginPercent < position.x) || (position.y < area.top) || (area.bottom*errMarginPercent < position.y)) {
+					return true;
+				}
+				return false;
+			};
+			var errMarginPercent = 1.01; // 1.01 is margin for Accumulated error. (Especially Edge, IE.)
+			var ratio = 0;
+			// going out from inner charArea?
+			if ((chartArea !== undefined)&&(isOutofChartArea(this._view, chartArea, errMarginPercent))&&(isOutofChartArea(this._model, chartArea, errMarginPercent))) {
```

Comment:
I think you should only need to check `this._model` here. At the end of the animation `this._view` will have the same values as `this._model`